src/components/calcite-tree-item/calcite-tree-item.scss

```diff
@@ -1,3 +1,17 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-tree-chevron-hover: the :hover color of tree-item's chevron icon
+ * @prop --calcite-tree-hover-line-width: the width of the vertical line on :hover in nested tree-items when tree has lines attribute
+ * @prop --calcite-tree-line: the color of the vertical line in nested tree-items when tree has lines attribute
+ * @prop --calcite-tree-line-hover: the color of the vertical line on :hover in nested tree-items when tree has lines attribute
+ * @prop --calcite-tree-line-width: the width of the vertical line in nested tree-items when tree has lines attribute
+ * @prop --calcite-tree-text: the color of tree-item font
```

Comment:
I feel like these should be handled with our set of brand, info?, grey css variables, so they’d be handled there. That might require a css refactor, but like we should use existing ones here.  Unsure about letting folks adjust width cc @bstifle ?

Reply:
Can you clarify which vars should not be exposed?

Reply:
@macandcheese @bstifle how about just exposing these for now?
```
  --calcite-tree-text: the color of tree-item font
  --calcite-tree-vertical-padding: the amount of top/bottom padding around each tree-item
```

Reply:
Thought @bstifle on these?  -  I'd suggest not showing either - tree text should be overridable with a global css var, and vertical padding seems weird to expose unless we had a specific request for it in the past?

Reply:
Gotcha, I'll remove these too @macandcheese

Reply:
done!